### ISSUE TITLE
Should be able to import an asset whose extension isn't css/js into vendor.css/vendor.js

### ISSUE THREAD

**Issue by lolmaus** [ember-cli/ember-cli#6614]
### TL/DR

This should work:

```js
  app.import('bower_components/yandex-maps/index.1', {
    outputFile: 'assets/vendor.js'
  })
```

Currently, this import statement does nothing.

### Motivation

Yandex Maps are a huge web service. Their maps and address DBs of the CIS region (which is 1/7 of the world's land) are way more precise than that of Google, Bing, etc.

Yandex Maps JS SDK is proprietary and only distributed via [direct URL](https://api-maps.yandex.ru/2.1/?lang=en_US). No GitHub, no npm package. The URL has no file extension.

The natural way to import he SDK into a frontend app is via `bower install yandex-maps=https://api-maps.yandex.ru/2.1/?lang=en_US`.

Unfortunately, Bower saves it as `bower_components/yandex-maps/index.1`. There is no simple way to tell Bower to add the `js` extension.

I'm sure Yandex Maps isn't the only example. **Ember CLI should be able to import JS/CSS files even if their extension isn't `js` or `css`.

PS I know about `<script src="...">`. That a valid but miserable workaround.

**Comment by boris-petrov** [comment]
I'm also trying to achieve the same thing - I have some `.es6` files that I want to `app.import` as JavaScript. Was this implemented at any point in `ember-cli`?

**Comment by Turbo87** [comment]
@boris-petrov unfortunately nobody has opened a pull request on this so far, but feel free to work on it. it seems that @lolmaus already started to work on it, but I don't see any related PRs.

**Comment by locks** [comment]
@lolmaus is this something you still want to pursue?

**Comment by lolmaus** [comment]
@locks Uhm, this was five years ago. 😳 No, I'm not gonna finalize that attempt now, sorry.

**Comment by locks** [comment]
I've been practicing necromancy once again xD
